sprint_tool/main.py

```diff
@@ -126,10 +137,20 @@ def get_values(field, f_name=None):
         for result in results:
             if result["status"] == "Success":
                 success += 1
+                if watchers:
+                    for watchman in watchers:
```

Comment:
Why not watcher? With the final season of Game of Thrones coming up I like the reference, but ladies weren't allowed to join the Nightswatch, but we do have excellent engineers that we work with who incidentally happen to be women also.

Reply:
fixed